tests/database_controller/cleanup.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker exec -i recipes-db_recipes-1 bash -c 'PGPASSWORD=password psql -h db_recipes -U djangouser -d djangodb -c "truncate auth_user cascade;"'
```

Comment:
In my local machine the database container name is `recipes_db_recipes_1`. but in CI its the same like above. A bit strange.